src/common/util/navigator_gpu.ts

```diff
@@ -70,9 +70,9 @@ export function getGPU(recorder: TestCaseRecorder | null): GPU {
     ): Promise<GPUAdapter | null> {
       const promise = oldFn.call(this, { ...defaultRequestAdapterOptions, ...options });
       if (recorder) {
-        void promise.then(async adapter => {
+        void promise.then(adapter => {
           if (adapter) {
-            const info = await adapter.requestAdapterInfo();
```

Comment:
We should make this robust to `.info` not being implemented yet, otherwise it will cause an unhandled promise rejection.

Though I wonder if this patching is entirely unnecessary now that we have `print_environment`? @greggman

Reply:
@kainino0x I'm now calling `requestAdapterInfo()` if `info` is not available.

Reply:
This is still useful. `print_environment` is a test that may not get run. If you set the `debug` option and run a single test case you'll see this info and you can see it change if you set the forced `powerPreference` option.